tools/SourceKit/lib/SwiftLang/SwiftLangSupport.h

```diff
@@ -24,12 +24,13 @@
 #include "swift/AST/DiagnosticConsumer.h"
 #include "swift/AST/PluginRegistry.h"
 #include "swift/Basic/ThreadSafeRefCounted.h"
+#include "swift/Frontend/PrintingDiagnosticConsumer.h"
```

Comment:
This would be an example of why it can be useful, though it probably doesn't really matter in this case (ie. the header is really only needed in the implementation, but now slows down everything including `SwiftLangSupport.h`)